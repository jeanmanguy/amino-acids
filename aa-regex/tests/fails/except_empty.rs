use aa_regex::*;

fn main() {
    let wrong = except!();
}
