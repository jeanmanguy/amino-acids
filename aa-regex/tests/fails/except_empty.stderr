error: no exceptions to add
 --> $DIR/except_empty.rs:4:17
  |
4 |     let wrong = except!();
  |                 ^^^^^^^^^
  |
  = help: add exceptions like `except!(P)` to avoid prolines for example
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected expression, found end of macro arguments
 --> $DIR/except_empty.rs:4:17
  |
4 |     let wrong = except!();
  |                 ^^^^^^^^^
